SpatialGDK/Source/SpatialGDKEditor/Private/SchemaGenerator/SpatialGDKEditorSchemaGenerator.cpp

```diff
@@ -398,7 +398,7 @@ TMap<uint32, FString> CreateComponentIdToClassPathMap()
 	return ComponentIdToClassPath;
 }
 
-void SaveSchemaDatabase()
+bool SaveSchemaDatabase()
```

Comment:
Is this the preferred way to return errors in Unreal? I'm assuming they don't use exceptions but maybe error codes? Output variables? Returning a bool seems very old school to me, but could be how things work in Unreal land.

Reply:
I was largely following the convention that the rest of the code there was demonstrating. I agree using bools perhaps isn't the most elegant approach.